background/background.js

```diff
@@ -208,6 +209,11 @@ contextMenus = {
     title: 'openOptions',
     click: browserCommands.openOptions,
   },
+  'reload': {
+    presentIf: () => localStorage.installType !== 'normal',
```

Comment:
AFAICT the very first install won't have this in localStorage because the type is determined asynchronously. It will appear on the next manual reload or after restarting the browser.

It can be fixed by moving createContextMenus outside of its block and declaring it as a normal function, then calling it in onInstalled. Didn't test it though.

```js
  chrome.management.getSelf(info => {
    localStorage.installType = info.installType;
    if (reason === 'install' && info.installType === 'development' && chrome.contextMenus) {
      createContextMenus(['reload']);
    }
  });
```

Reply:
>AFAICT the very first install won't have this in localStorage

Hmm. I did check beforehand. Fresh profile, fresh side-load, check in the console directly after loading: `localStorage.installType === 'development'`.

Reply:
By the time you open devtools the type is already determined, but I meant the moment when `contextMenus =` is executed. You can verify my guess by adding `console.log('type', localStorage.installType)` right before.

Reply:
Yep, undefined. Guess I'll try yours.

Reply:
That's what you meant, right? Seems to work as expected.

Reply:
Essentially yes but I suggested converting `const createContextMenus = ids => {` to `function createContextMenus(ids) {` and move it a few lines upwards so there will be no need to shift the other code around (the normal functions are usable even before they're declared due to *hoisting*).

Reply:
![spiderman](https://user-images.githubusercontent.com/14100003/74311871-6bcb0600-4d3e-11ea-8902-698ec3c065bf.jpg)

Reply:
That's up to you.

Reply:
You and your Jedi mind tricks. =) Better now?

Reply:
Yeah I guess. It's an idiosyncrasy I probably got in childhood after reading Azimov's The End of Eternity, which impressed me with the concept of finding the minimal necessary influence to change the outcome.

Reply:
Aah, someday I'll do it right the first time. =)